### PR TITLE
Running Docker Compose on Windows

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- QPPCT-718

### Changes proposed in this PR:
- Updates the docker file to be able to run on Docker-toolbox windows 7
    - When I ran docker before the changes, I received the same error the user was getting.
    - Unable to test on windows 10 as docker doesn't work on a windows vm. (Hyper V issues.)
- I excluded our main docker file because I doubt any users would be running our spunk in their docker.

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [n/a] New unit tests written to cover new functionality.
- [n/a] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [n/a] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: DockerfileTest [hunk 1]
```diff
@@ -10,6 +12,7 @@ WORKDIR /usr/src/app/
 RUN mvn install -Dmaven.test.skip -Djacoco.skip=true > /dev/null
 RUN cp ./rest-api/target/rest-api.jar /usr/src/run/
 RUN cp -r ./tools/docker/docker-test-artifacts/* /usr/src/run/
+RUN dos2unix /usr/src/run/qppConverterTest.sh
```
Comment (halprin) [inline DockerfileTest]: Wait a second, that file already has LF line endings.  AKA, it already has "UNIX line endings".  Is it possible that the person on Windows is using some weird setting on `git` that forces CRLF line endings?

File: DockerfileTest [hunk 2]
```diff
@@ -2,6 +2,8 @@ FROM maven:3-jdk-8
 
 RUN mkdir -p /usr/src/app/
 RUN mkdir -p /usr/src/run/
+RUN apt-get update && apt-get install
+RUN apt-get install dos2unix
```
Comment (halprin) [inline DockerfileTest]: You shouldn't need this separate line.  You can move `dos2unix` to the above line.

File: DockerfileTest (hunk 1 above)
Comment (sfradkin) [reply DockerfileTest]: Even so, maybe it's best to just keep this in here.  At worst it won't change the file.

File: DockerfileTest (hunk 2 above)
Comment (saquino0827) [reply DockerfileTest]: I only separated this for readability.

File: DockerfileTest (hunk 1 above)
Comment (saquino0827) [reply DockerfileTest]: @halprin yes that' what is happening... but I figured we can keep this so the user has little to change when running our docker compose test.

File: DockerfileTest (hunk 2 above)
Comment (halprin) [reply DockerfileTest]: Then you don't need the `apt-get install` on the first line.  Although, I believe the preferred method is to put the `apt-get update` and all installs on the same line.

Comment (saquino0827) [reply DockerfileTest]: 👍

**Pull request merged**